solr/contrib/prometheus-exporter/conf/grafana-solr-dashboard.json

```diff
@@ -176,7 +176,7 @@
           "refId": "A"
         },
         {
-          "expr": "count(count by (node_name) (solr_collections_replica_state))",
+          "expr": "count(count by (node_name) (solr_collections_replica_state)) < solr_collections_live_nodes or solr_collections_live_nodes",
```

Comment:
This is a "bugfix" for when you e.g. have a cluster with 3 nodes but no configured collections. Instead of showing 0, we then fall back to live_nodes if that has a bigger number.